inferno-core/src/Inferno/Instances/Arbitrary.hs

```diff
@@ -227,7 +232,7 @@ deriving instance ToADTArbitrary Fixity
 
 deriving via (GenericArbitrary Fixity) instance Arbitrary Fixity
 
-instance Arbitrary pos => Arbitrary (Comment pos) where
+instance (Arbitrary pos) => Arbitrary (Comment pos) where
```

Comment:
Sorry the rest of the changes in this file are formatting changes like this one... think my ormulu version changed or something